HW7/src/js/index.js

```diff
@@ -0,0 +1,30 @@
+function newTask() {
+    let inputValue = document.getElementById("add-task").value;
+    if (inputValue === "") {
+        return
```

Comment:
You can add an alert with some text. For example, `your task is empty` or `value can't be empty` or smth like this.